core/src/main/java/nl/nn/adapterframework/util/StreamUtil.java

```diff
@@ -69,28 +70,38 @@ public static OutputStream getOutputStream(Object target) throws IOException {
 			return (OutputStream) target;
 		} 
 		if (target instanceof String) {
-			String filename=(String)target;
-			if (StringUtils.isEmpty(filename)) {
-				throw new IOException("target string cannot be empty but must contain a filename");
-			}
-			try {
-				return new FileOutputStream(filename);
-			} catch (FileNotFoundException e) {
-				FileNotFoundException fnfe = new FileNotFoundException("cannot create file ["+filename+"]");
-				fnfe.initCause(e);
-				throw fnfe;
+			return getFileOutputStream((String)target);
```

Comment:
This is a bit tricky. Could you please make the method `getOutputStream()` and `getWriter()` deprecated?